.github/workflows/ci.yaml

```diff
@@ -69,8 +69,7 @@ jobs:
   integration-test:
     name: Integration test charm
     needs:
-      - lint
-      - unit-test
+      # - unit-test
```

Comment:
@juditnovak we need to revert this

Reply:
Yes, this was a leftover, by mistake. Thx 4 spotting it :-)